common/src/main/java/com/revenuecat/purchases/common/caching/DeviceCache.kt

```diff
@@ -38,8 +38,8 @@ open class DeviceCache(
     val attributionCacheKey = "$SHARED_PREFERENCES_PREFIX.attribution"
     val tokensCacheKey: String by lazy { "$SHARED_PREFERENCES_PREFIX$apiKey.tokens" }
 
-    private val purchaserInfoCachesLastUpdatedCacheBaseKey: String by lazy {
-        "$SHARED_PREFERENCES_PREFIX$apiKey.purchaserInfoLastUpdated"
+    private val customerInfoCachesLastUpdatedCacheBaseKey: String by lazy {
+        "$SHARED_PREFERENCES_PREFIX$apiKey.customerInfoLastUpdated"
```

Comment:
*I think* this should remain the same, otherwise we lose the current cache.

Reply:
👍  was gonna consult on that...

Reply:
In the iOS side, I left all the keys the same. It would require a migration if we changed them. IMO that's out of scope 😄

Reply:
although, i think all this will result in is a force fetch the first time...

Reply:
If the app starts offline, there won't be caches either, so it will be unusable. We should keep this as is or build a migration path which I think it's overkill